wazo_calld/plugins/calls/tests/test_services.py

```diff
@@ -104,3 +106,42 @@ def test_direction_of_call_to_who_is_caller(self):
         call = self.services.channel_destroyed_event(event)
 
         assert_that(call.is_caller, equal_to(True))
+
+    def _assert_conversation_direction(self, directions, expected):
+        assert_that(
+            self.services._conversation_direction_from_directions(directions),
+            equal_to(expected),
+        )
+
+    def test_call_direction(self):
+
+        inbound_channel = 'inbound'
+        outbound_channel = 'outbound'
+        internal_channel = 'internal'
+        unknown_channel = 'unknown'
+
+        self._assert_conversation_direction([], internal_channel)
```

Comment:
Suggestion: I would find this clearer to read:
```python
direction = self.services._conversation_direction_from_directions
assert_that(direction([internal, internal]), equal_to(internal))
assert_that(direction([inbound, internal]), equal_to(inbound))
...
```

Reply:
I agree. Done :smile: